README.md

```diff
@@ -1969,8 +1969,15 @@ My name is [Ruslan](https://github.com/BlackTowerKing). Glad to be here.
 
 ![yoda](assets/images/yoda-kitty.png)
 
----
```

Comment:
could you please revert changes here?    It necessary, because in this case we will have all these changes which are absent to other student's versions of a README (which are still at review), and at their files will be conflicts

Reply:
My latest commit reverts the changes in question. Please, review.